src/popup/utils/constants.js

```diff
@@ -195,3 +194,17 @@ export const IDENTICON_SIZES = {
   normal: 38,
   lg: 64,
 };
+
+export const CONFIRM_POPUP_TX_FIELDS = [
```

Comment:
Does it have sense to move it to `SignDetailsList` as far as it is used only there?